### ISSUE TITLE
AuRa Consensus

### ISSUE THREAD

**Issue by crossoveranx** [hyperledger/besu#1338]
### Description
As a blockchain developer, it would be great to integrate the AuRa consensus mechanism to support enterprise blockchains that utilize this algorithm. The OpenEthereum client has had some issues since the transition from Parity -> OpenEthereum, and I think that many users are looking for alternative clients.

**Comment by timbeiko** [comment]
Agreed that AuRa support would be great! Unfortunately, it's not on the PegSys (or other maintainers') roadmap in the short term, but if anyone wanted to contribute an implementation, we'd be happy to help with a codebase overview and other support required by implementers 😄

**Comment by iFA88** [comment]
> ### Description
> 
> As a blockchain developer, it would be great to integrate the AuRa consensus mechanism to support enterprise blockchains that utilize this algorithm. The OpenEthereum client has had some issues since the transition from Parity -> OpenEthereum, and I think that many users are looking for alternative clients.

I think you still use poanetwork openethereum backport for it: https://github.com/poanetwork/openethereum-backport-3.1

**Comment by matt-nelson-csi** [comment]
Close as GC will not need AuRav going forward.
